integration_tests/tests/web_app_certos.rs

```diff
@@ -505,6 +503,71 @@ mod workroom {
       d.wait_for_text("#create-request-container > div > div > div > p", r"template-show*").await;
     }
 
+    integration_test!{ shows_usage_statistics_for_issuances (c, d)
+      let mut chain = TestBlockchain::new().await;
+      let alice = c.alice().await;
+      signup_and_verify(&d, &c.site).await;
+      create_wizard(&d, &c.site, 5, "testing-template").await;
+      sign_wizard(&d).await;
+      chain.fund_signer_wallet();
+      chain.simulate_stamping().await;
+      for i in 1..6 {
```

Comment:
¿Por que se tienen que crear los download proof links acá y no están creados ya? 
No iteramos colecciones por rango, es confuso después al leerlo entender el sentido de la coleccion que se está iterando. Mejor traerse el Request que creamos con el wizard e iterarle los entries.

Reply:
No están creados. sin embargo cambié el código por el siguiente:

  for entry in c.site.request().find(&1).await?.entry_vec().await? {
    entry.in_signed()?.try_complete().await?;
  }
Así, iteramos sobre los entries de este request y llamamos directamente a la función que es ejecutada por el worker que crea estos download_proof_link